pkg/agent/baker_test.go

```diff
@@ -18,80 +19,161 @@ func generateTestData() bool {
 	return os.Getenv("GENERATE_TEST_DATA") == "true"
 }
 
-var _ = Describe("Assert generated customData and cseCmd", func() {
-	DescribeTable("Generated customData and CSE", func(folder, k8sVersion string, configUpdator func(*datamodel.NodeBootstrappingConfiguration)) {
-		cs := &datamodel.ContainerService{
-			Location: "southcentralus",
-			Type:     "Microsoft.ContainerService/ManagedClusters",
-			Properties: &datamodel.Properties{
-				OrchestratorProfile: &datamodel.OrchestratorProfile{
-					OrchestratorType:    datamodel.Kubernetes,
-					OrchestratorVersion: k8sVersion,
+func getFakeContainerService(k8sVersion string) *datamodel.ContainerService {
+	cs := &datamodel.ContainerService{
```

Comment:
Do we need to add network plugin to azurecni and loadbalancer to standard since Windows only can be enabled when these conditions are met?

Reply:
We used azure cni plugin and standard lb.
```
					LoadBalancerSku:      "Standard",
					CustomHyperkubeImage: "mcr.microsoft.com/oss/kubernetes/hyperkube:v1.16.15-hotfix.20200903",
					ClusterSubnet:        "10.240.0.0/16",
					NetworkPlugin:        "azure",
```